src/components/BMDashboard/MaterialsList/SelectForm.jsx

```diff
@@ -4,7 +4,7 @@ export default function SelectForm({ materials, setSelectedProject, setSelectedM
   // create selectable projects
   let projectsSet = [];
   if (materials.length) {
-    projectsSet = [...new Set(materials.map(mat => mat.project.name))];
+    projectsSet = [...new Set(materials.map(mat => mat.project?.name))];
```

Comment:
**nit**:  Would `uniqueProjects` or similar be a better name?  It's actually an array and I feel like this is  tending towards Hungarian notation.

I see others like materials are doing similar.